remoting/zookeeper/client_test.go

```diff
@@ -18,14 +18,17 @@
 package zookeeper
 
 import (
-	"fmt"
 	"testing"
 	"time"
 )
 
 import (
 	"github.com/dubbogo/go-zookeeper/zk"
-	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
I think u can rollback the testify/require to testify/assert package.

Reply:
## an example

```go
func TestSomething(t *testing.T) {
	var err error

	err = perrors.Errorf("something1")
	assert.NoError(t, err)

	err = perrors.Errorf("something2")
	assert.NoError(t, err)
}
```

- use `testify/assert` will output two errors on console:

```
=== RUN   TestSomething
--- FAIL: TestSomething (0.00s)
    client_test.go:151: 
        	Error Trace:	client_test.go:151
        	Error:      	Received unexpected error:
        	            	something1
        	            	github.com/apache/dubbo-go/remoting/zookeeper.TestSomething
        	            		D:/GoPath/src/github.com/apache/dubbo-go/remoting/zookeeper/client_test.go:150
        	            	testing.tRunner
        	            		D:/Go/src/testing/testing.go:909
        	            	runtime.goexit
        	            		D:/Go/src/runtime/asm_amd64.s:1357
        	Test:       	TestSomething
    client_test.go:154: 
        	Error Trace:	client_test.go:154
        	Error:      	Received unexpected error:
        	            	something2
        	            	github.com/apache/dubbo-go/remoting/zookeeper.TestSomething
        	            		D:/GoPath/src/github.com/apache/dubbo-go/remoting/zookeeper/client_test.go:153
        	            	testing.tRunner
        	            		D:/Go/src/testing/testing.go:909
        	            	runtime.goexit
        	            		D:/Go/src/runtime/asm_amd64.s:1357
        	Test:       	TestSomething
FAIL
``` 

- use `testify/require` just output the first error and stop the test case intermediately

## why I prefer `testify/require`

- Because errors in test case may have some dependency: first error causes second error, then I think it's better to use `tesify/require`, which only shows the first error.
- Using `tesify/assert` may output too many errors, but developers only need to focus on fixing the first error.  

@AlexStocks

Reply:
However, it is impossible that we test the same error case in one Testxxx.

Reply:
@AlexStocks Ok, I revert `require` to `assert`.